### PR TITLE
Refactoring (repo, composer, doctrine)

### PR DESCRIPTION
**Pull request by f3l1x**

### CODE REVIEW

**Comment by f3l1x** [comment]
Ready for review. @paveljanda

File: .scrutinizer.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ tools:
 
 filter:
   excluded_paths:
-    - docs/*
+    - .docs/*
```
Comment (paveljanda) [inline .scrutinizer.yml]: 👍

File: src/DataSource/DoctrineCollectionDataSource.php [hunk 2]
```diff
@@ -4,9 +4,10 @@
 
 namespace Ublaboo\DataGrid\DataSource;
 
-use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Collection;
 use Doctrine\Common\Collections\Criteria;
+use Doctrine\Common\Collections\Selectable;
+use LogicException;
```
Comment (paveljanda) [inline src/DataSource/DoctrineCollectionDataSource.php]: Could you please refer to native classes directly in code and not in the use statement? Like `\LogicException`. Reason is, there are many vendors using the same class name as the native one (for example, `Dibi\DateTime`) and thus when you use '\DateTime`, you can see what class it is right away.

Than you. 👍

Comment (f3l1x) [reply src/DataSource/DoctrineCollectionDataSource.php]: Yup

**Pull request merged**